README.md

```diff
@@ -0,0 +1,136 @@
+<h1 align="center">
+  <br>
+  <a href="https://www.fury.gl"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/fury-logo.png" alt="FURY" width="200"></a>
+  <br>Free Unified Rendering in Python<br>
+
+</h1>
+
+<h4 align="center">A software library for scientific visualization in Python.
+</h4>
+
+<p align="center">
+  <a href="https://travis-ci.com/fury-gl/fury">
+    <img src="https://travis-ci.com/fury-gl/fury.svg?branch=master" alt="Fury Travis">
+  </a>
+  <a href="https://dev.azure.com/fury-gl/fury/_build/latest?definitionId=1&branchName=master"><img src="https://dev.azure.com/fury-gl/fury/_apis/build/status/fury-gl.fury?branchName=master">
+</a>
+<a href="https://pypi.python.org/pypi/fury"><img src="https://img.shields.io/pypi/v/fury.svg"></a>
+<a href="https://anaconda.org/conda-forge/fury"><img src="https://anaconda.org/conda-forge/fury/badges/version.svg"></a>
+<a href="https://codecov.io/gh/fury-gl/fury"><img src="https://codecov.io/gh/fury-gl/fury/branch/master/graph/badge.svg"></a>
+<a href="https://app.codacy.com/app/fury-gl/fury?utm_source=github.com&utm_medium=referral&utm_content=fury-gl/fury&utm_campaign=Badge_Grade_Dashboard"><img src="https://api.codacy.com/project/badge/Grade/922600af9f94445ead5a12423b813576"></a>
+
+</p>
+
+<p align="center">
+  <a href="#general-informations">General Informations</a> •
+  <a href="#key-features">Key Features</a> •
+  <a href="#installation">Installation</a> •
+  <a href="#how-to-use">How to use</a> •
+  <a href="#credits">Credits</a> •
+  <a href="#contribute">Contribute</a>
+
+</p>
+
+|         |         |         |
+|:--------|:--------|:--------|
+| <a href="#"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/ws_smaller.gif" alt="FURY Networks" width="400px"></a> | <a href="#"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/swarming_simulation.gif" alt="swarming simulation" width="400px"></a> | <a href="#"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/shaders_horse.gif" alt="shaders horse" width="400px"></a> |
+| *Network Visualization*          | *Swarming/flocking simulation based on simple boids rules*  |  *Easy shader effect integration.*  |
+| <a href="#"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/shaders_sdf.gif" alt="sdf" width="400px"></a>  | <a href="#"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/collides_simulation.gif" alt="Collides simulation" width="400px"></a> | <a href="#"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/physics_bricks.gif" alt="Physics bricks" width="400px"></a> |
+| Ray Marching and Signed Distance Functions |  *Advanced Nano simulation capablities.* | *Interoperability with the [pyBullet](https://pybullet.org/wordpress/) library.*  |
```

Comment:
Just say Particle collisions. Shading of the box of particle collisions looks strange. Assign Nasim to fix this and regenerate the gif.